### ISSUE TITLE
Element is blocked if CSS class name is `addarea`

### ISSUE THREAD

**Issue by ddomgn** [smartadblock/SmartAdBlock#2]
Example:
```html
<html>
<head>
    <title>Test</title>
    <style>
        .addarea {
            display: block !important;
        }
    </style>
</head>
<body>
    <p class="addarea">Hello, World!</p>
</body>
</head>
```
Expected: a page with "Hello, World" shown on it.

Actual: empty page.

The `add` term may mean addition for example. Maybe we need more deliberate filtering of CSS classes.

**Comment by smartadblock** [comment]
Hi

addarea class is part of easylist:
[...]
##.add_baner
##.add_topbanner
##.addarea
##.addarearight
##.addbanner
[...]
For now smartadblock is using easylist. Maybe we could switch to a better maintained one, do you know a better maintained blocking list?

**Comment by ddomgn** [comment]
> [...]
> For now smartadblock is using easylist. Maybe we could switch to a better maintained one, do you know a better maintained blocking list?

No, I don't. So it's a drawback of using ad blocker, not a SmartAdBlock bug.

Thank you.
